sympy/geometry/point.py

```diff
@@ -442,32 +557,44 @@ def __add__(self, other):
 
         """
 
-        if iterable(other) and len(other) == len(self):
-            return Point([simplify(a + b) for a, b in zip(self, other)])
-        else:
-            raise ValueError(
-                "Points must have the same number of dimensions")
+        if iterable(other):
+            coords = [simplify(a + b) for a, b in zip(self, other)]
+            if len(coords) != self.ambient_dimension:
+                raise ValueError("Points must have the same number of dimensions")
+            if all(isinstance(c, Number) for c in coords):
```

Comment:
here and below, I'm not sure this is a good idea...can you explain the rationale of `Point(.1,.2) -> Point(1/10, 1/5)` but `Point(1,2)*0.1 -> Point(0.1, 0.2)`?